### ISSUE TITLE
Calendar updated

### ISSUE THREAD

**Issue by overath2** [noQ-sweden/noq#703]

**Comment by patrik-pwned** [comment]
@overath2  Ska jag lägga in alla [event](https://drive.google.com/file/d/10T5xTVJlaVmzQ3O7dacSK-I8eyKRNuSf/view) för respektive team som styrelse, P&C, SM/PM osv i google kalender?

Scrum-eventen är inlagda i var fall.

**Comment by overath2** [comment]
Nej låt teamlead lägga in för sina team. Men allt annat vore toppen?

**Comment by patrik-pwned** [comment]
Ja, PO:ornas event är inlagda liksom scrum eventen. Nu är det upp till övriga
